packages/css/src/breadcrumb/breadcrumb.scss

```diff
@@ -0,0 +1,73 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+@import "../../utils/breakpoint";
+
+@mixin reset {
+  box-sizing: border-box;
+  break-after: avoid;
+  break-inside: avoid;
+  margin-block: 0;
+  margin-inline: 0;
+  padding-inline: 0;
+}
+
+.amsterdam-breadcrumb {
+  color: var(--amsterdam-breadcrumb-color);
+  font-family: var(--amsterdam-breadcrumb-font-family, inherit);
+  font-size: var(--amsterdam-breadcrumb-font-size-narrow);
+  line-height: var(--amsterdam-breadcrumb-line-height);
+  text-transform: lowercase;
+
+  @media screen and (width > $amsterdam-breakpoint) {
+    font-size: var(--amsterdam-breadcrumb-font-size-wide);
+  }
+}
+
+.amsterdam-breadcrumb-list {
+  column-gap: 0.5rem;
+  display: flex;
+  flex-wrap: wrap;
+  list-style: none;
+
+  @include reset;
+}
+
+.amsterdam-breadcrumb-item {
+  align-items: center;
+  display: flex;
+}
+
+.amsterdam-breadcrumb-item:not(:last-child)::after {
+  background-color: var(--amsterdam-breadcrumb-color);
+  content: "";
+  display: inline-block;
+  font-size: 16px;
+  height: 16px;
+  margin-inline-start: 1rem;
+  -webkit-mask-image: url("../../../../proprietary/icons/static/ChevronRight.svg");
+  mask-image: url("../../../../proprietary/icons/static/ChevronRight.svg");
+  -webkit-mask-size: 16px;
+  mask-size: 16px;
+  width: 16px;
+}
+
+.amsterdam-breadcrumb-link {
+  color: inherit;
+  cursor: pointer;
+  outline-offset: var(--amsterdam-breadcrumb-item-link-outline-offset);
+  text-decoration: none;
+
+  &:focus,
+  &:hover,
+  &:active {
+    box-shadow: var(--amsterdam-breadcrumb-item-link-hover-box-shadow);
```

Comment:
Shouldn't we use `text-decoration: underline` for this? Similar to the link component?

Reply:
I think you can not increase the space between the underline and the text. Also I think this is a different implementation of a link then the actual `<Link />` component

Reply:
done